src/.clang-tidy

```diff
@@ -19,8 +19,10 @@ performance-*,
 -performance-avoid-endl,
 -performance-enum-size,
 -performance-inefficient-string-concatenation,
+-performance-inefficient-vector-operation,
```

Comment:
Given that https://github.com/bitcoin/bitcoin/pull/31305 fixes the issue, are we disabling it as a temporary measure?

Reply:
Right.

It depends on which PR is merged first :)